### PR TITLE
8230492: font-family not set in HTMLEditor if font name has a number in it

### PR DESCRIPTION
**Pull request by arun-Joseph**
In the HTMLEditor, when positioning the caret in a text and trying to set a font-family that has a number in it is not working.

Issue: In [CSSPropertyParser.cpp](https://github.com/openjdk/jfx/blob/master/modules/javafx.web/src/main/native/Source/WebCore/css/parser/CSSPropertyParser.cpp), concatenateFamilyName() function parses only identifiers. So, when a number is introduced in a font-name, it fails.

Fix: Pass the font-name as a string in HTMLEditorSkin.java by adding quotes.

A new font is added as a resource for the test. This font is same as modules/javafx.web/src/main/native/Tools/DumpRenderTree/fonts/WebKit Layout Tests 2.ttf

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

## Issue
[JDK-8230492](https://bugs.openjdk.java.net/browse/JDK-8230492): font-family not set in HTMLEditor if font name has a number in it

## Approvers
 * Kevin Rushforth ([kcr](@kevinrushforth) - **Reviewer**) **Note!** Review applies to 5a1fbadee8c2862fcafeca4928b08bf44678d520
 * Hadzic Samir ([shadzic](@Maxoudela) - Author)

### CODE REVIEW

**Comment by Maxoudela** [comment]
Will this be in conflict with https://github.com/openjdk/jfx/pull/12 ? I'll look at your unit tests in order to implement one similar

**Comment by kevinrushforth** [comment]
I would think that the two fixes are independent of one another, but it would be a good idea to test HTMLEditor with both patches applied.

**Comment by arun-Joseph** [comment]
I tested HTMLEditor with both patches applied, and both fixes are working fine without any conflict.

File: tests/system/src/test/java/test/javafx/scene/web/HTMLEditorTest.java [hunk 1]
```diff
@@ -294,4 +303,55 @@ public void checkStyleProperty() throws Exception {
...
+            });
+
+            htmlEditor.setHtmlText("<body>Sample Text</body>");
+
+            webView.focusedProperty().
+                addListener((observable, oldValue, newValue) -> {
+                if (newValue) {
+                    ComboBox<String> fontFamilyComboBox = null;
+                    int i = 0;
+                    for(Node comboBox : htmlEditor.lookupAll(".font-menu-button")) {
```
Comment (kevinrushforth) [inline tests/system/src/test/java/test/javafx/scene/web/HTMLEditorTest.java]: Please add a space after the `for` keyword.

**Pull request closed without merging**